content/guides/kubernetes/velero-gs.md

```diff
@@ -0,0 +1,389 @@
+    ```bash
+    velero backup describe $BACKUP
+
+    velero backup logs $BACKUP
+    ```
+
+## 🔥Wreak Havoc🔥
+Now that we have a happy cluster state and that most excellent blog you created...let’s nuke Ghost. 
+
+Let’s say that _**someone**_ (you) “accidentally” (definitely intentionally) deleted the Ghost Helm chart and the persistent volume claim (PVC) by running the following:
+```bash
+helm delete --namespace ghost ghost
+kubectl -n ghost delete pvc data-ghost-mariadb-0
+```
+Try to connect to Ghost again. Shoot... Now the blog site is down and so is that _**super important**_ blog post you just made. What now?
+
+## Restore
+Well, it’s a good thing you have backups! Now it's time to look into restoring from a backup.
+
+1. To see what all you can do with restore run:
+    ```bash
+    velero restore -h
+    ```
+    
+2. Let’s now wait for all of the resources to be gone. First check the output of `get all`:
+    ```bash
+    kubectl -n ghost get all
+    ```
+    And the Ghost and MariaDB persistent volumes claims (PVCs) as well:
+    ```bash
+    kubectl get pvc
+    ```
+    > Note: It might take a minute until everything is properly deleted; so you may run these commands above until they give you a (somewhat ominous) `No resources found` output.
+
+2. Now to perform a restore:
+   
+   To take a look at what backups you have you can run the following again:
+    ```bash
+    velero backup get
+    ```
+   To restore from the specific backup we created first:
+    ```bash
+    velero restore create --include-namespaces ghost --from-backup $BACKUP
+    ```
+
+3. So the big questions are:
+   * Did all the resources come back up and therefore the Ghost site?
+   * Is the _super important_ blog back, or only the default blogs?
+
+   * Let’s find out. First let’s check to see if our resources are back and running.
+    ```bash
+    kubectl -n ghost get all
+
+    NAME                         READY   STATUS    RESTARTS   AGE
+    pod/ghost-86b56c9d44-gk2f8   1/1     Running   0          2m13s
+    pod/ghost-mariadb-0          1/1     Running   0          2m13s
+
+    NAME                    TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)        AGE
+    service/ghost           LoadBalancer   10.108.14.225   34.105.103.109   80:30353/TCP   2m12s
+    service/ghost-mariadb   ClusterIP      10.108.5.14     <none>           3306/TCP       2m12s
+
+    NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/ghost   1/1     1            1           2m12s
+
+    NAME                               DESIRED   CURRENT   READY   AGE
+    replicaset.apps/ghost-86b56c9d44   1         1         1       2m12s
+
+    NAME                             READY   AGE
+    statefulset.apps/ghost-mariadb   1/1     2m12s
+    ```
+
+    Again, wait until all the pods above are `Running` with counts of `1/1` pods in `READY` state.
+
+4. Now get the EXTERNAL_IP from your service and go to the IP in your web browser:
```

Comment:
this may be different from the original external IP as there is no guarantee the cloud provider won't have re-used the previous one.  How do I verify and fix the ghost install to ensure it continues to work (not a big deal if using real dns obviously, but since we're using IP addresses here.)